chipsec/logger.py

```diff
@@ -30,7 +30,8 @@
 """
 Logging functions
 """
-
+import logging as pyLogging
+import coloredlogs
```

Comment:
I have a feeling this is causing the CI build problem.  You may want to just submit the logging changes and leave the color coding alone for now.  We can then look at updating the color logging.